src/loaders/README.md

```diff
@@ -39,6 +39,7 @@ an animation and its duration:
 | repetitions       | Infinity | Number of times to play the clip, in addition to the first play. Repetitions are ignored for `loop: once`. |
 | timeScale         | 1        | Scaling factor for playback speed. A value of 0 causes the animation to pause. Negative values cause the animation to play backwards. |
 | clampWhenFinished | false        | If true, halts the animation at the last frame. |
+| startFrame        | 0        | Sets the animation to a specific time (in milliseconds). This is useful when you need to jump to an exact time in an animation. The input parameter will be scaled by the mixer's timeScale. |
```

Comment:
Note that the description makes it sound like this will move a running animation to the given time, whereas the property name and implementation appear to only affect a newly-started animation. I think the behavior looks good but would just clarify the description wording.

Reply:
I have updated the description to more accurately reflect the functionality.

Reply:
Looks good to me!